docs/deploy.md

```diff
@@ -0,0 +1,65 @@
+
+If the import statement is not used and instead the src of the image was hard coded like:
+```
+<img src="assets/concord.png"/>
+```
+Webpack has no control of this, so at runtime this will be loaded relative to the html file.  So when the `index.html` is at the top level, the browser will look for `https://collaborative-learning.concord.org/assets/concord.png` and not find it. So hard coded paths like this should be converted to using import statements.
+
+In some cases we dynamically compute a path to load an asset from. In most of these places webpack imports can still be used. Webpack supports this by static analysis of the import function, so we just need to change those places in the code slightly. Here is the documentation about this:
+https://webpack.js.org/api/module-methods/#dynamic-expressions-in-import
+
+If using import is too difficult you can work around this by using the special `__webpack_public_path__` variable. In CLUE there is a `getAssetUrl` function to make this easy. Like this:
+
+```
+<img src={getAssetUrl("assets/concord.png")}/>
+```
+A possible reason for doing this is if you are working with an external library that you don't have control over and need to pass it a path to load an asset.
+
+When possible, switching to an import is preferred because it means that webpack knows about all of the referenced assets. This means we can use webpack to build a manifest which is useful for offline support.
+
+Note: there is a `publicPath` configuration option for the `HtmlWebpackPlugin`. This is a different but related option, it controls the prefix the plugin adds before assets (javascript and css) referenced in the generated html file. This option is used so the `index-top.html` references assets in the version folder and `index.html` references assets in the same folder.
+
+## Local testing for compatibility with index-top.html
+
+When running in the regular dev server, you won't see errors when using hard coded paths. 
+
+Typically, hard coded paths will only work if you are using `CopyWebpackPlugin`. This is because these assets need to be copied into the `dist` folder. With import statements the assets are copied for you. If you remove the `CopyWebpackPlugin` you will likely see errors when using the dev server, so you can find the places that need to fixed.
```

Comment:
```suggestion
Typically, hard coded paths will only work if you are using `CopyWebpackPlugin`. This is because these assets need to be copied into the `dist` folder. With import statements the assets are copied for you. If you remove the `CopyWebpackPlugin` you will likely see errors when using the dev server, so you can find the places that need to be fixed.
```